TemperatureMeter.c

```diff
@@ -0,0 +1,154 @@
+/*
+htop - TemperatureMeter.c
+(C) 2013 Ralf Stemmer
+(C) 2014 Blair Bonnett
+(C) 2020 Maxim Kurnosenko
+Released under the GNU GPL, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "TemperatureMeter.h"
+
+#include "ProcessList.h"
+#include "CRT.h"
+#include "StringUtils.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <limits.h>
+#include <errno.h>
+#include <sensors/sensors.h>
+
+static int get_cpu_sensors_amount(void) {
+   int n = 0, count = 0;
+
+   for (const sensors_chip_name *chip = sensors_get_detected_chips(NULL, &n); chip; chip = sensors_get_detected_chips(NULL, &n)) {
+      char buffer[64];
+      if (sensors_snprintf_chip_name(buffer, sizeof(buffer), chip) < 0)
+         continue;
+
+      // cpu_thermal is for Raspberry Pi sensor support
+      if ((String_startsWith(buffer, "coretemp") == 0) && (String_startsWith(buffer, "cpu_thermal") == 0))
```

Comment:
`String_startsWith` is defined as a macro that already checks for `== 0` on the return value of `strncmp`, thus returning `bool`. Thus this line better would read `!String_startsWith(...)` or `String_startsWith(...) == false`.